.github/CODEOWNERS

```
@@ -48,6 +48,7 @@
 /packages/adyen-utils @bigcommerce/team-integrations
 /packages/affirm-integration @bigcommerce/team-integrations
 /packages/afterpay-integration @bigcommerce/team-integrations
+/packages/amazon-pay @bigcommerce/team-integrations
```

It might be better to stick to main pattern and add `-integration`:
```suggestion
/packages/amazon-pay-integration @bigcommerce/team-integrations
```

In this case might be easier to regenerate package